### PR TITLE
Add release plan to contribution guide

### PR DESCRIPTION
**Pull request by paulirish**
I think this accounts for most of the release tasks.

Made it rather copypasteable, rather than too heavy on prose.

Feel free to edit the PR directly. :)

### CODE REVIEW

**Comment by paulirish** [comment]
> WDYT about just checking this script in?

most of the important stuff is manual for now, so i think it wouldn't work great as a `yarn run ship-release` just yet. but hopefully eventually!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,3 +19,51 @@ accept your pull requests.
...
+
+
+## Release guide
+
+```sh
+# * Build the latest *
+yarn
+cd lighthouse-cli; yarn; yarn run build; cd ..
+cd lighthouse-viewer; yarn; gulp; cd ..
+cd lighthouse-extension; yarn; gulp build; cd ..
```
Comment (brendankenny) [inline CONTRIBUTING.md]: extension needs to be built after bumping the version (`gulp package` just packages, doesn't rebuild)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -19,3 +19,51 @@ accept your pull requests.
...
+
+# * Bump it *
+echo "Bump the versions in extension/app/manifest.json and package.json"
+
+# * Test err'thing * 
+echo "Test the CLI."
+yarn run smoke
+echo "Test a fresh local install (see below)"
+echo "Test the extension"
+echo "Test the lighthouse-viewer build"
```
Comment (brendankenny) [inline CONTRIBUTING.md]: what's the best way to do this locally? @ebidel

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -19,3 +19,51 @@ accept your pull requests.
...
+
+# * Put up the PR *
+echo "Branch and commit the version bump."
+echo "Generate a PR and get it merged."
+
+# * Deploy-time *
+cd lighthouse-extension; gulp package; cd ..
```
Comment (brendankenny) [inline CONTRIBUTING.md]: or could just add a `gulp build` in here

File: CONTRIBUTING.md (hunk 1 above)
Comment (paulirish) [inline CONTRIBUTING.md]: done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ebidel) [inline CONTRIBUTING.md]:
```
cd lighthouse-viewer
gulp build
# start a server in dist/ and open that location in a tab. You'll see the viewer.
# Drop in a .json or paste an existing gist url (https://gist.github.com/ebidel/b9fd478b5f40bf5fab174439dc18f83a).
# Check for errors.



```

Comment (paulirish) [inline CONTRIBUTING.md]: chill. feel free to toss that into the doc if you want. :)

**Pull request merged**